airflow/models/renderedtifields.py

```diff
@@ -153,6 +154,23 @@ def _redact(self):
         for field, rendered in self.rendered_fields.items():
             self.rendered_fields[field] = redact(rendered, field)
 
+    @classmethod
+    @internal_api_call
+    @provide_session
+    def update_runtime_evaluated_template_fields(
+        cls, ti: TaskInstance, session: Session = NEW_SESSION
+    ) -> None:
+        """Update rendered task instance fields for cases where runtime evaluated, not templated."""
```

Comment:
can we make it "private" by either adding `:meta private:` or leading underscore?

(i think best to avoid making _anything_ public unless really necessary)

Reply:
That totally makes sense. Thanks. Adjusted.